api/v1beta1/metallb_webhook.go

```diff
@@ -36,18 +37,18 @@ func (metallb *MetalLB) SetupWebhookWithManager(mgr ctrl.Manager) error {
 var _ webhook.Validator = &MetalLB{}
 
 // ValidateCreate implements webhook.Validator so a webhook will be registered for MetalLB.
-func (metallb *MetalLB) ValidateCreate() error {
-	return metallb.validate()
+func (metallb *MetalLB) ValidateCreate() (admission.Warnings, error) {
+	return admission.Warnings{}, metallb.validate()
```

Comment:
Let's expand this to `if err := metallb.validate(); err != nil` form

Reply:
Sure! is this what you mean?
```
func (metallb *MetalLB) ValidateCreate() (admission.Warnings, error) {
	if err := metallb.validate(); err != nil {
		return admission.Warnings{}, err
	}

	return admission.Warnings{}, nil
}
```

Reply:
yes